### PR TITLE
Mostly Revert "adding support for specified log "message" / type"

### PR DESCRIPTION
**Pull request by abernix**
Reverts apollographql/router#2797

There's some discussion about the change here — including the API and the ergonomics of the output — that we'd like to be a bit more intentional about and see if there's another way we can accomplish this.

### CODE REVIEW

**Review by o0Ignition0o — APPROVED** [review]

File: examples/logging/rhai/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Rhai script
 
-Demonstrates basic logging using Rhai.
+Demonstrates extracting body details and header manipulation via Rhai script.
```
Comment (garypen) [inline examples/logging/rhai/README.md:3]: We should keep that change. It was a driveby fix
```suggestion
Demonstrates basic logging using Rhai.
```

**Review by garypen — CHANGES_REQUESTED** [review]
